File: Assignment2/Assignment2/Problem4.c
// Accept two number from user and display first number in second number of time.


#include<stdio.h>

void Display(int iNo, int iFrequency)
{
	int Int =0;
	
	if (iFrequency < 0)
	{
		
		iFrequency= -iFrequency;
		
	}
	
	for(Int=1;Int<=iFrequency;Int++)
	{
		printf("%d\n",iNo);
		
	}
	
	
}

int main()
{
	int iValue =0;
	int iCount =0;
	
	printf("enter Number\n");
	scanf("%d",&iValue);
	
	printf("enter frequency\n");
	scanf("%d",&iCount);
	
	Display(iValue,iCount);
	
	return 0;
	
}

File: Assignment 5/Problem4.java
// Accept number from user and display Sum of Non factors 

import java.lang.*;
import java.util.*;

class Maths
{
	public int iValue=0;
	
	public Maths(int iNo)
	{
		this.iValue=iNo;
	}
	
	public int SumNonFactor()
	{
		int iCnt =0;
		int iSum =0;
		
		if(iValue<0)
		{
			iValue = -iValue;
		}
		
		for(iCnt=1;iCnt<iValue;iCnt++)
		{
			if(iValue%iCnt != 0)
			{
				iSum = iSum+iCnt;
			}
		}
		return iSum;
	}
}

class Assignment4
{
	
	public static void main(String arr[])
	{
		int iNumber =0;
		int iRet =0;
		
		System.out.println("Enter Number");
		
		Scanner sobj = new Scanner(System.in);
		iNumber = sobj.nextInt();
		
		Maths mobj = new Maths(iNumber);
		
		iRet = mobj.SumNonFactor();
		
		System.out.println("Sum of NonFator is : "+iRet);
	}
	
}